docs/references/javascript/organization/membership-request.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Organization Membership Request
```

Comment:
The other pages in the JS references will be updated in an upcoming JavaScript Docs Revamp. But, essentially, we are using sentence-casing for headings ( see https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#15-use-sentence-case-for-titles )

```suggestion
title: Organization membership requests
```